tck/methods/account.go

```diff
@@ -0,0 +1,279 @@
+		}
+		transaction.SetKey(key)
+	}
+	if accountParams.InitialBalance != 0 {
+		transaction.SetInitialBalance(hedera.HbarFromTinybar(accountParams.InitialBalance))
+	}
+	if accountParams.ReceiverSignatureRequired {
+		transaction.SetReceiverSignatureRequired(accountParams.ReceiverSignatureRequired)
+	}
+	if accountParams.MaxAutomaticTokenAssociations != 0 {
+		transaction.SetMaxAutomaticTokenAssociations(accountParams.MaxAutomaticTokenAssociations)
+	}
+	if accountParams.StakedAccountId != "" {
+		accountId, err := hedera.AccountIDFromString(accountParams.StakedAccountId)
+		if err != nil {
+			return nil, jrpc2.Errorf(-32603, "Internal error", ErrorStatus{Status: err.Error()})
+		}
+		transaction.SetStakedAccountID(accountId)
+	}
+	if accountParams.StakedNodeId != 0 {
+		transaction.SetStakedNodeID(accountParams.StakedNodeId)
+	}
+	if accountParams.DeclineStakingReward {
+		transaction.SetDeclineStakingReward(accountParams.DeclineStakingReward)
+	}
+	if accountParams.AccountMemo != "" {
+		transaction.SetAccountMemo(accountParams.AccountMemo)
+	}
+	if accountParams.AutoRenewPeriod != 0 {
+		transaction.SetAutoRenewPeriod(accountParams.AutoRenewPeriod)
+	}
+	if accountParams.PrivateKey != "" {
+		key, err := hedera.PrivateKeyFromString(accountParams.PrivateKey)
+		if err != nil {
+			return nil, jrpc2.Errorf(-32603, "Internal error", ErrorStatus{Status: err.Error()})
+		}
+
+		_, err = transaction.FreezeWith(a.sdkService.Client)
+		if err != nil {
+			return nil, jrpc2.Errorf(-32603, err.Error())
+		}
+		transaction.Sign(key)
+	}
+
+	txResponse, err := transaction.Execute(a.sdkService.Client)
+	if err != nil {
+		fmt.Println(err)
+		return nil, jrpc2.Errorf(-32603, "Internal error").WithData(ErrorStatus{Status: err.Error()})
+	}
+	receipt, err := txResponse.GetReceipt(a.sdkService.Client)
+	var accId string
+	if receipt.Status == hedera.StatusSuccess {
+		accId = receipt.AccountID.String()
+	}
+
+	return &AccountResponse{AccountId: accId, Status: receipt.Status.String()}, nil
+}
+
+// CreateAccountFromAlias Create an account from aliasId by transferring  some HBAR amount from given account (opperatorId)
+func (a *AccountService) CreateAccountFromAlias(_ context.Context, fromAliasParams AccountFromAliasParams) (*AccountResponse, error) {
+	operator, err := hedera.AccountIDFromString(fromAliasParams.OperatorId)
+	alias, err := hedera.AccountIDFromString(fromAliasParams.AliasAccountId)
+
+	if err != nil {
+		return nil, jrpc2.Errorf(-32603, "Internal error", ErrorStatus{Status: err.Error()})
+	}
+
+	resp, err := hedera.NewTransferTransaction().SetGrpcDeadline(&threeSec).
+		AddHbarTransfer(operator, hedera.NewHbar(float64(fromAliasParams.InitialBalance)).Negated()).
+		AddHbarTransfer(alias, hedera.NewHbar(float64(fromAliasParams.InitialBalance))).
+		Execute(a.sdkService.Client)
+
+	if err != nil {
+		fmt.Println(err)
+		return nil, jrpc2.Errorf(-32603, "Internal error").WithData(ErrorStatus{Status: err.Error()})
```

Comment:
In some places `WithData` is used in some places it is done like:
```
if err != nil {
		return nil, jrpc2.Errorf(-32603, "Internal error", ErrorStatus{Status: err.Error()})
```

Reply:
Made it so we return the error and the errorMiddleware handles the specific error logic.